agent/agent.go

```diff
@@ -85,8 +85,8 @@ type Config struct {
 	// KeyFile specifies the path to TLS certificate key file
 	KeyFile string
 
-	// RPC address of local serf node.
-	SerfRPCAddr string
+	// SerfConfig specifies serf client configuration.
```

Comment:
Is it ok to nest configs?

Reply:
You mean embed config into one another? Perfect use of aggregation imo.